mapper/mapper.go

```diff
@@ -102,7 +104,7 @@ func updateNamespace(namespace *corev1.Namespace, dkList *dynatracev1beta1.DynaK
 	return updated, nil
 }
 
-func updateLabels(matches bool, dynakube *dynatracev1beta1.DynaKube, namespace *corev1.Namespace) (bool, error) {
+func updateLabels(matches bool, dynakube *dynatracev1beta1.DynaKube, namespace *corev1.Namespace, log logr.Logger) (bool, error) {
```

Comment:
since this function has a lot of parameters now, consider refactoring some or all of them into a struct and using that as a parameter

the same or a similar struct should also be used for the `updateNamespace` function, as it too has a lot of paramteres

Reply:
I'm not a fan of creating structs just to pass them as variables, you are just moving the bloat to a different place.

furthermore it kinda asks the question "why isn't this a method of the struct ?"

Reply:
Imho this would be a little overkill, since I would need two structs one for `updateNamespace` and one for `update Labels`. Futhermore I think it would add more complexity to the code, just for a shorter parameter list.